packages/plugin/src/estree-parser/utils.ts

```diff
@@ -69,37 +66,19 @@ export function extractCommentsFromAst(loc: Location): Comment[] {
   let token = loc.startToken;
 
   while (token !== null) {
-    const { kind, value, line, column, start, end, next } = token;
-    if (kind === TokenKind.COMMENT) {
-      comments.push({
-        type: 'Block',
-        value,
-        loc: {
-          start: { line, column },
-          end: { line, column },
```

Comment:
this is a mistake, `end` is always as `start` I fixed it by `column - 1 + (end - start)` where `end` is `range[1]` and start is `range[0]`.
> Note: `-1` is needed because ESLint treat columns as 0 based